sda-commons-dependencies/build.gradle

```diff
@@ -40,16 +42,16 @@ dependencies {
     api "org.mongodb:mongo-java-driver:3.10.1"
 
     // Kafka
-    api "org.apache.avro:avro:1.8.2"
-    api "org.apache.kafka:kafka_2.12:1.1.1"
-    api "org.apache.kafka:kafka-clients:1.1.1"
+    api "org.apache.avro:avro:1.9.2"
+    api "org.apache.kafka:kafka_2.12:$kafkaVersion"
+    api "org.apache.kafka:kafka-clients:$kafkaVersion"
     api "org.apache.zookeeper:zookeeper:3.4.10"
-    api "io.confluent:kafka-avro-serializer:4.1.2"
-    api "io.confluent:kafka-schema-registry:4.1.2"
-    api "io.confluent:kafka-schema-registry-client:4.1.2"
-    api "io.confluent:common-config:4.1.2"
-    api "io.confluent:common-utils:4.1.2"
-    api "io.confluent:rest-utils:4.1.2"
+    api "io.confluent:kafka-avro-serializer:$confluentVersion"
+    api "io.confluent:kafka-schema-registry:$confluentVersion"
+    api "io.confluent:kafka-schema-registry-client:$confluentVersion"
+    api "io.confluent:common-config:$confluentVersion"
```

Comment:
Im not sure, but can we use a jetty bom here?
https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-bom

Reply:
Seems to be a good idea. Dropwizard does not use the jetty-bom:
https://github.com/dropwizard/dropwizard/blob/release/1.3.x/dropwizard-bom/pom.xml
I'll check it out!

Reply:
Yes, it works! :)